controller/controller_server/csi_controller_server.py

```diff
@@ -97,7 +97,7 @@ def CreateVolume(self, request, context):
                 else:
                     logger.debug("volume found : {}".format(volume))
 
-                    if not source_id and volume.capacity_bytes != request.capacity_range.required_bytes:
+                    if not source_id and volume.capacity_bytes < request.capacity_range.required_bytes:
                         message = "Volume was already created with different size."
```

Comment:
btw, regardless of this PR, I think it would be better to include the numbers in the error message

Reply:
done